src/resources/PDF-healthHygiene/questions-health-hygiene/question-6/no.html

```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-info" role="alert">
```

Comment:
can't this be sym linked for when its the same text, this is exactly like q4/yes.html etc

Reply:
We have avoided sim linking and our full forms are bytes only.